src/github/github-event-handler.ts

```diff
@@ -68,4 +69,13 @@ export class GitHubEventHandler {
       },
     });
   }
+
+  async getToken(installationId: number) {
+    const auth = createAppAuth({
+      appId: this._appId,
+      privateKey: this._privateKey,
+    });
+    const token = await auth({ type: "installation", installationId });
+    return token.token;
+  }
```

Comment:
So I assume that this is the kernel logic to generate the token with the intent to pass it along to the plugin.